### PR TITLE
Adding consequence column

### PR DESCRIPTION
**Pull request by zheins**
Adding a field in the annotated record for VEP consequence_terms, called Consequences.

@angelicaochoa @sheridancbio

### CODE REVIEW

File: annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java [hunk 1]
```diff
@@ -168,6 +168,7 @@ public AnnotatedRecord annotateRecord(MutationRecord record, boolean replaceHugo
                 additionalProperties.get("Protein_Position") != null ? additionalProperties.get("Protein_Position") : "",
                 additionalProperties.get("Codons") != null ? additionalProperties.get("Codons") : "",
                 additionalProperties.get("Hotspot") != null ? additionalProperties.get("Hotspot") : "",
+                additionalProperties.get("Consequence") != null ? additionalProperties.get("Consequcne") : "",
```
Comment (angelicaochoa) [inline annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java]: typo here

File: annotator/src/main/java/org/cbioportal/models/AnnotatedRecord.java [hunk 2]
```diff
@@ -58,7 +59,8 @@
         HEADER.add("Protein_position");
         HEADER.add("Codons");        
         HEADER.add("Hotspot");
-    }       
+        HEADER.add(HEADER.indexOf("Variant_Classification"), "Consequence");
```
Comment (angelicaochoa) [inline annotator/src/main/java/org/cbioportal/models/AnnotatedRecord.java]: We want to replace the column `Variant_Classification` with `Consequence`? I thought we just wanted to add the `Consequence` column?

Comment (zheins) [inline annotator/src/main/java/org/cbioportal/models/AnnotatedRecord.java]: Add does not replace the element, it sticks in at that position and moves all other elements back one index.

File: annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java (hunk 1 above)
Comment (zheins) [inline annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java]: thanks!

File: annotator/src/main/java/org/cbioportal/models/AnnotatedRecord.java (hunk 2 above)
Comment (angelicaochoa) [inline annotator/src/main/java/org/cbioportal/models/AnnotatedRecord.java]: thanks :)

**Pull request merged**